src/AdvancedHMC.jl

```diff
@@ -29,7 +29,7 @@ end
 # θ: position variables / model parameters
 # r: momentum variables
 
-include("adaptation/Adaptation.jl")
+include("Adaptation/Adaptation.jl")
```

Comment:
Maybe use a lower case name for a directory, which is easier to navigate on terminals.